server/src/models/Expense.py

```diff
@@ -0,0 +1,16 @@
+from pydantic import BaseModel
+
+# Location is stored as an x and y coordinate (easier to input into maps APIs)
+# All fields are currently required
+class Expense(BaseModel):
+    id: int
```

Comment:
I would change this to a custom type for compatibility with MongoDB. See this for more info: https://www.mongodb.com/developer/quickstart/python-quickstart-fastapi/#database-models
```suggestion
    id: PyObjectId = Field(default_factory=PyObjectId, alias="_id")
    # .... the rest of the fields
    class Config:
        allow_population_by_field_name = True
        arbitrary_types_allowed = True
        json_encoders = {ObjectId: str}
```